src/app/cli/src/app.rs

```diff
@@ -173,10 +187,28 @@ pub async fn run(workspace_layout: WorkspaceLayout, args: cli::Cli) -> Result<()
         let cli_catalog = configure_cli_catalog(&final_base_catalog, is_multi_tenant_workspace)
             .add_value(current_account.to_current_account_subject())
             .build();
+        let maybe_server_catalog = if cli_commands::command_needs_server_components(&args) {
+            let server_catalog =
+                configure_server_catalog(&final_base_catalog, is_multi_tenant_workspace)
+                    .add_value(current_account.to_current_account_subject())
+                    .build();
+
+            Some(server_catalog)
+        } else {
+            None
+        };
 
-        (guards, final_base_catalog, cli_catalog, output_config)
+        (
+            guards,
+            final_base_catalog,
+            cli_catalog,
+            maybe_server_catalog,
+            output_config,
+        )
     };
 
+    initialize_base_components(&base_catalog).await?;
```

Comment:
I've recently added logging to the original call.
There's no error printed anywhere, if this fails, i.e. due to database connectivity issue.

Reply:
I've also added logging for this method as well -- at this point, it's a common problem since logging is initiated far from immediately in this (`run()`) method

This is scheduled to be fixed in
- https://github.com/kamu-data/kamu-cli/issues/845